simd/src/Kokkos_SIMD.hpp

```diff
@@ -25,7 +25,8 @@
 #include <Kokkos_SIMD_AVX2.hpp>
 #endif
 
-#ifdef KOKKOS_ARCH_AVX512XEON
+// FIXME_NVHPC nvc++ doesn't seem to support AVX512.
+#if !defined(KOKKOS_COMPILER_NVHPC) && defined(KOKKOS_ARCH_AVX512XEON)
```

Comment:
Should we instead only define `KOKKOS_ARCH_AVX512XEON` when not using NVC++?

Reply:
I don't feel strongly about it either way. [`c229f2e` (#6223)](https://github.com/kokkos/kokkos/pull/6223/commits/c229f2eeabcdf829e05b3fb476505e95ebe1fcea) changes it.